### ISSUE TITLE
Select & Upload Crashes Browser

### ISSUE THREAD

**Issue by thekcool** [dessant/search-by-image#58]
**System** 

* OS name: [e.g. Windows, Ubuntu] Windows 10 Pro 
* OS version: [e.g. 10] 1903 18362.295
* Browser name: [e.g. Chrome, Firefox] Waterfox 
* Browser version: [e.g. 60] 56.2.13 (64 bit)
* Extension version: [e.g. 1.0.0] 1.21.2

**Bug description**

Using "Select & Upload" in Context Menu or Browser Toolbar crashes Waterfox.

**Logs**

```
// REPLACE WITH LOGS
```

**Comment by dessant** [comment]
I cannot reproduce the issue, try capturing the logs during the crash. It's also possible that this is a browser bug that should be reported to the Waterfox team.

**Comment by grahamperrin** [comment]
[Search By Image addon crashes WF : waterfox](https://www.reddit.com/r/waterfox/comments/e9npnu/search_by_image_addon_crashes_wf/) 

tl;dr

* not reproducible by me
* troubleshooting steps are suggested.
